tests/apps/sequencing_metrics_parser/conftest.py

```diff
@@ -72,6 +42,26 @@ def fixture_test_lane() -> int:
     return 1
 
 
+@pytest.fixture(name="bcl_convert_reads_for_test_sample")
+def fixture_bcl_convert_reads_for_test_sample() -> int:
+    return 15962796
+
+
+@pytest.fixture(name="bcl_convert_test_q30_bases_percent")
```

Comment:
I don't like this, to hard code a calculated value and use it in a test to verify a result. Better to calculate the value in the fixture instead or calculate it here with values from the fixture.

Reply:
It is not calculated in this parser - it comes directly from the output